presto-spi/src/main/java/com/facebook/presto/spi/type/SqlVarbinary.java

```diff
@@ -67,4 +67,28 @@ public boolean equals(Object obj)
         SqlVarbinary other = (SqlVarbinary) obj;
         return Arrays.equals(bytes, other.bytes);
     }
+
+    @Override
+    public String toString()
+    {
+        StringBuilder builder = new StringBuilder();
+        int i = 0;
+        for (byte b : bytes) {
+            if (i != 0) {
+                if (i % 32 == 0) {
+                    builder.append("\n");
```

Comment:
No, you should avoid anything that introduces non deterministic behavior, especially in a toString method.